hasura/metadata/databases/default/tables/public_vaults.yaml

```diff
@@ -1,6 +1,6 @@
 table:
-  schema: public
   name: vaults
+  schema: public
```

Comment:
Seems a bunch of these changes are just re-ordering the fields... would be preferable to keep that change out of here, or do it first in a separate PR.

Reply:
seems like it was done automatically by Hasura CLI. Related issue: https://github.com/hasura/graphql-engine/issues/8743

will revert in this PR for now

Reply:
@crabsinger I think there are a bunch of hasura files that need to be regenerated since the upgrades